spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/PropertiesMeterFilter.java

```diff
@@ -66,7 +68,8 @@ private static MeterFilter createMapFilter(Map<String, String> tags) {
 
 	@Override
 	public MeterFilterReply accept(Meter.Id id) {
-		boolean enabled = lookup(this.properties.getEnable(), id, true);
+		boolean enabled = lookup(this.properties.getEnable(), id, true,
+				(all) -> this.properties.getEnable().getOrDefault(all, null));
```

Comment:
The defaults are out of sync here. One it `true` and the other is `null`. This means we're missing a test as, with these changes, the following test will fail with a `NullPointerException`:

```java
@Test
public void acceptWhenHasNoMatchingEnabledPropertyShouldReturnNeutral() {
	PropertiesMeterFilter filter = new PropertiesMeterFilter(
			createProperties("enable.something.else=false"));
	assertThat(filter.accept(createMeterId("spring.boot")))
			.isEqualTo(MeterFilterReply.NEUTRAL);
}
```